strategy/live_trader.py

```
@@ -594,6 +609,201 @@ def cancel_all_orders(self) -> Dict[str, Any]:
+                is_filled = (mstate.order_status_exit_up == "FILLED")
+                if not is_filled and mstate.order_id_exit_up:
+                    client = self.get_clob_client()
+                    if client:
+                        try:
+                            ord_info = client.get_order(mstate.order_id_exit_up)
+                            st = (ord_info.get("status") or "").upper()
+                            sz = float(ord_info.get("size_matched", 0.0) or 0.0)
+                            if st in ("MATCHED", "FILLED") or sz >= self.shares:
+                                is_filled = True
+                                mstate.order_status_exit_up = "FILLED"
+                        except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)